src/pages/registration/SameNameWarning.tsx

```diff
@@ -28,10 +29,7 @@ export const SameNameWarning = ({ name }: SameNameWarningProps) => {
       <Link
         target="_blank"
         data-testid={dataTestId.registrationLandingPage.duplicateRegistrationSearchLink}
-        to={{
-          pathname: UrlPathTemplate.Home,
-          search: `?query=${encodeURIComponent(name)}`,
-        }}>
+        to={`${UrlPathTemplate.RegistrationNew}/${id}`}>
```

Comment:
Har en hjelpemetode fra `urlPaths.ts` for å hente lenke til registrering
```suggestion
        to={getRegistrationLandingPagePath(id)}>
```

Reply:
That's useful, thanks! (Fixed)